LICENSE.md

```diff
@@ -1,25 +1,30 @@
-The Lingua-Franca toolkit is licensed under the BSD 2-Clause License:
+BSD 2-Clause License
 
-Copyright (c) 2019-2020, Industrial Cyberphysical Systems (iCyPhy). All rights reserved.
+Copyright (c) 2019-present, The Lingua Franca authors:
 
-Redistribution and use in source and binary forms, with or without modification, 
-are permitted provided that the following conditions are met:
+<a href="https://github.com/lf-lang/lingua-franca/graphs/contributors">
```

Comment:
If we want to stay in control of these links, we should create aliases for them, like:
```suggestion
<a href="https://contrib.lf-lang.org">
```